content/v2.0/write-data/sample-data/demo-data.md

```diff
@@ -0,0 +1,60 @@
+---
+title: InfluxDB Cloud demo data
+description: >
+  Use **InfluxDB Cloud** demo data buckets for quick access to different types of sample data.
+menu:
+  v2_0:
+    name: Demo data
+    parent: Sample data
+weight: 201
+v2.0/tags: [sample data]
+products: [cloud]
+---
+
+Use **InfluxDB Cloud** demo data buckets for quick and easy access to different
+types of demo data that let you explore and familiarize yourself with InfluxDB Cloud.
+
+- [Add a demo data bucket](#add-a-demo-data-bucket)
+- [Explore demo data](#explore-demo-data)
+- [Visualize demo data](#visualize-demo-data)
+- [Demo data pricing and restrictions](#demo-data-pricing-and-restrictions)
+
+## Demo data sets
+Choose from the following demo data sets:
+
+- **Website Monitoring**  
+  Explore, visualize, and monitor HTTP response metrics from InfluxData websites.
+
+_More demo data options coming soon._
+
+## Add a demo data bucket
+1.  In the navigation menu on the left, click **Data (Load Data)** > **Buckets**.
+
+    {{< nav-icon "data" >}}
+
+2.  Click **{{< icon "plus" >}} Add Demo Data** and select the demo data bucket to add.
+3.  The Demo Data bucket appears in your list of buckets.
+
+## Explore demo data
+Use the [Data Explorer](https://v2.docs.influxdata.com/v2.0/visualize-data/explore-metrics/)
+to query and visualize data in demo data buckets.
+
+In the navigation menu on the left, click **Explore (Data Explorer)**.
+
+{{< nav-icon "explore" >}}
+
+## Visualize demo data
```

Comment:
bc folks can also "visualize" in Data Explorer, how about combining "Explore and visualize demo data", and then say do one of the following:
- Open Data Explorer
- Open Dashboard